idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/phe.py

```diff
@@ -0,0 +1,801 @@
+        self.cold_side = ControlVolume0DBlock(default={
+            "dynamic": self.config.dynamic,
+            "has_holdup": self.config.has_holdup,
+            "property_package": self.config.cold_side.property_package,
+            "property_package_args": self.config.cold_side.property_package_args})
+
+        self.cold_side.add_state_blocks(has_phase_equilibrium=False)
+
+        self.cold_side.add_material_balances(
+            balance_type=self.config.material_balance_type,
+            has_mass_transfer=False,
+            has_phase_equilibrium=False,
+            has_rate_reactions=False)
+
+        self.cold_side.add_momentum_balances(
+            balance_type=self.config.momentum_balance_type,
+            has_pressure_change=self.config.has_pressure_change)
+
+        # ======================================================================
+        # Add Ports to control volumes
+        # hot-side
+        self.add_inlet_port(name="hot_inlet",
+                            block=self.hot_side, doc='inlet Port')
+        self.add_outlet_port(name="hot_outlet",
+                             block=self.hot_side, doc='outlet Port')
+
+        # cold-side
+        self.add_inlet_port(name="cold_inlet",
+                            block=self.cold_side, doc='inlet Port')
+        self.add_outlet_port(name="cold_outlet",
+                             block=self.cold_side, doc='outlet Port')
+        # ======================================================================
+        # Add performace equation method
+        self._make_params()
+        self._make_performance_method()
+
+    def _make_params(self):
+        self.P = Param(initialize=self.config.passes,
+                       units=None,
+                       doc="Total number of passes for hot or cold fluid")
+        self.PH = RangeSet(self.P,
+                           doc="Set of hot fluid passes")
+        self.PC = RangeSet(self.P,
+                           doc="Set of cold fluid passes(equal to PH)")
+        self.plate_thermal_cond = Param(initialize=plate_param.kp,
+                                        units=pyunits.W / pyunits.m / pyunits.K,
+                                        doc="Plate thermal conductivity")
+        self.plate_thick = Param(initialize=plate_param.bp,
+                                 units=pyunits.m,
+                                 doc="Plate thickness")
+        self.plate_gap = Param(initialize=plate_param.b,
+                               units=pyunits.m,
+                               doc="Plate gap")
+        self.plate_length = Param(initialize=plate_param.L,
+                                  units=pyunits.m,
+                                  doc="Plate length ")
+        self.plate_width = Param(initialize=plate_param.w,
+                                 units=pyunits.m,
+                                 doc="Plate width ")
+        self.plate_area = Param(initialize=plate_param.Ap,
```

Comment:
Should this be a separate Param? Shouldn't this be calculated from the length and width instead?

If not , ca comment would help to prevent others asking the same question.